.github/workflows/code-cover-publish.yaml

```diff
@@ -56,3 +56,7 @@ jobs:
           parent: false
           destination: 'crl-codecover-public/pr-cockroach/'
           process_gcloudignore: false
+
+      - name: 'Call DeadManSnitch'
+        run: |
+          curl -X GET 'https://nosnch.in/c2d75963ee' -d 'message=Code coverage uploaded to GCS'
```

Comment:
No auth/API key is needed for this? Wondering if the url should be a secret in that case - although admittedly I haven't looked into how DeadManSnitch works.

Reply:
No auth/API key needed.

About it being a secret, I kinda wondered the same thing, but we already have one [publicly available in the repo](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachprod/docker/entrypoint.sh#L20), and the worst thing that can happen is that something reports the status in lieu of the action (granted that it defeats the purpose...).

Maybe @cockroach-dev-inf has an opinion?

Reply:
We don't typically have to keep these URL's secret -- the risk isn't huge.

If you want to make this one a secret, be my guest (although you've technically already "leaked" this one, so you should "rotate" it :) )

I'm not aware of any broad issue with third parties hitting these URL's and as mentioned the worst thing that can happen is we think the thing has reported a success when it hasn't.